.gitignore

```diff
@@ -1,2 +1,7 @@
+<<<<<<< HEAD
+node_modules
+server
+=======
 /node_modules/
 .env
+>>>>>>> 6d9ec3d020c78f13aa4d116448035f7095945698
```

Comment:
You can't push changes with merge conflicts like that, you need to fix them
In that specific case you should remove everything, leaving server, node_modules, .env